concepts/Testing/Testing.md

```diff
@@ -97,6 +103,24 @@ describe('UsersController', function() {
 
 });
 ```
+## Running tests
+
+In order to run your test using mocha, you'll have to use `mocha` in the command line and then pass as arguments the test your want to run, be sure to call bootstrap.test.js before the rest of your test like this `mocha test/bootstrap.test.js test/unit/**/*.test.js`
+
+#### Using `npm test` to run your test
+
+To avoid typing the mocha command, like stated before (specially when calling bootstrap.test.js) and using `npm test` instead, you'll need to modify your package.json.On the scripts obj, add a `test` key and type this as its value `./node_modules/.bin/mocha test/bootstrap.test.js test/unit/**/*.test.js` like this:
+
+```js
+ // Some config....
+ scripts": {
+    "start": "node app.js",
+    "debug": "node debug app.js",
+    "test": "./node_modules/.bin/mocha test/bootstrap.test.js test/unit/**/*.test.js"
```

Comment:
instead of `./node_modules/.bin/mocha test/bootstrap.test.js test/unit/**/*.test.js` you can use `mocha test/bootstrap.test.js test/unit/**/*.test.js`, since mocha is usually installed globally?